packages/ui-charts/assets/metadataQA.json

```diff
@@ -0,0 +1,448 @@
+    "doanyotherorganizationsfundthecreationormaintenanceofthisdataset": "Yes",
+    "whatorganizationfundedthecreationormaintenanceofthisdataset2": "Counties: Multnomah, Clackamas, Washington",
+    "whattypeoflegalentityisthefundingorganization2": "Government Agency",
+    "howisthefundingorganizationfunded2": "taxes / fees.",
+    "doanyotherorganizationsfundthecreationormaintenanceofthisdataset2": "No",
+    "whatisthisdatasetprimarilyusedfor": "from their website: \"For mapping, property analysis, and planning. Care should be taken in the use and interpretation of these data. Not all columns are fully populated by the assessor and the data are standardized for consistency across the data layer.\"",
+    "whatshouldntthisdatasetbeusedfor": "?",
+    "isthisdatasetusedforanyadditionalpurposes": "Yes",
+    "whatisthisdatasetadditionallyusedfor": "It's a public dataset that I'm sure people find all sorts of uses for. ",
+    "isthisdatasetusedforanyadditionalpurposes2": "No",
+    "doesthisdatasetrelyonotherdatasources": "No",
+    "hasthedatacollectionandpreprocessingbeendocumentedbythecreatorsofthedataset": "Yes",
+    "linktomethodsdocumentationfromdatasetcreator": "[from 2018 housing team] https://drive.google.com/drive/u/2/folders/1fPOdrI4vL9BD8DwvLQBQ9y-yULgSGRnI",
+    "howwasthedatacollected": "By the 2018 Housing team",
+    "ifthedatawascapturedautomatically": "was not. ",
+    "ifpeoplecollectedthedata": "counties collect data and share with metro",
+    "istherecollectedinformationmissingfromthedatasetandwhy": "unclear. There are complementary datasets on actual property taxes owed and paid that we have been hoping to get, but aren't by default part of this dataset. ",
+    "whatimportantinformationwasntcollected": "earlier data was just for Portland and then just for Multnomah county. ",
+    "whatdatavalidationhasbeenperformed": "na",
+    "whoperformedthedatavalidation": "na",
+    "isthedatasetasampleofinstancesfromalargerset": "No",
+    "arethereknowndataqualityissues": "Yes",
+    "whataretheknowndataqualityissues": "Some miscoding of property codes; possible projection issue for 1997 data. ",
+    "doesthisdatasetrelatetopeople": "No",
+    "whatdataprotectionstandardsdoesthisdatasetcomplywith": "don't know",
+    "doesthedatasetcontaininformationthatmightbeconsideredsensitiveorconfidential": "Yes",
+    "whatinformationdoesthedatasetcontainthatmightbeconsideredsensitiveorconfidential": "it's public, so not confidential, but includes names and addresses of owners of each lot in the region; and information about sale dates and prices. ",
+    "doesthedatasetcontaininformationthatmightbeconsideredinappropriateoroffensive": "No",
+    "emailaddress": "stephen.osserman@hackoregon.org",
+    "whatisyourteamwithintheorganization": "Housing",
+    "whatisyourorganization": "Hack Oregon - 2019 Project Season",
+    "whoareyou": "Stephen Osserman, Nick Kobel, Xander Blair"
+  }
+}
```

Comment:
Do these metadata json files need to be in both `ui-cards` and `ui-charts`?

Reply:
Apparently there was a whole unused assets folder in `ui-charts` being useless...